__tests__/feature/editing/editingLiteral.test.js

```diff
@@ -35,7 +35,7 @@ describe("editing a literal property", () => {
     screen.getByTestId("Remove foo")
     // There is language button.
     expect(screen.getByTestId("Change language for foo")).toHaveTextContent(
-      "English"
+      "en"
```

Comment:
Now displaying the code instead of the label next to the input, as the label may be really, really long. However, the label is available on mouseover.